Examples/RegistrationITKv3/CMakeLists.txt

```diff
@@ -79,13 +83,13 @@ target_link_libraries(ITKv3MultiResImageRegistration2 ${ITK_LIBRARIES})
 
 add_executable(ITKv3MultiResImageRegistration3 MultiResImageRegistration3.cxx )
 target_link_libraries(ITKv3MultiResImageRegistration3 ${ITK_LIBRARIES})
-
+if (ITKFEMRegistration_ENABLED)
```

Comment:
This is not necessary.  These are  100% redundant with ITKv4 registration examples.